airbyte-config/config-persistence/src/test/java/io/airbyte/config/persistence/StreamResetPersistenceTest.java

```diff
@@ -8,49 +8,25 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.Mockito.spy;
 
-import io.airbyte.db.factory.DSLContextFactory;
-import io.airbyte.db.factory.DataSourceFactory;
-import io.airbyte.db.factory.FlywayFactory;
-import io.airbyte.db.instance.configs.ConfigsDatabaseMigrator;
-import io.airbyte.db.instance.configs.ConfigsDatabaseTestProvider;
-import io.airbyte.db.instance.development.DevDatabaseMigrator;
-import io.airbyte.db.instance.development.MigrationDevHelper;
 import io.airbyte.protocol.models.StreamDescriptor;
-import io.airbyte.test.utils.DatabaseConnectionHelper;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.UUID;
-import org.jooq.SQLDialect;
-import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-class StreamResetPersistenceTest extends BaseDatabaseConfigPersistenceTest {
+class StreamResetPersistenceTest extends BaseConfigDatabaseTest {
```

Comment:
Changes in this class:
* Remove unneeded db setup, let `BaseConfigDatabaseTest`
* Remove usage of `dslContext` and replace it with `database`.